packages/providers/audit-logs-local/lib/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const auditLogContentType = require('./content-types/audit-log');
+
+const provider = {
+  async register({ strapi }) {
+    strapi.container.get('content-types').add('admin::', { 'audit-log': auditLogContentType });
+
+    // Return the provider object
+    return {
+      async saveEvent(event) {
+        // Rewrite userId key to user
+        const auditLog = { ...event, user: event.userId };
+        delete auditLog.userId;
+
+        // Save to database
+        await strapi.entityService.create('admin::audit-log', { data: auditLog });
+
+        return this;
+      },
+    };
```

Comment:
To avoid checking if the provider was registered all the time, I returned the useful methods from the bootstrap. This way if we have access to saveEvent, we know the provider has been registered already.

I found out it's what's done for the [upload providers](https://github.com/strapi/strapi/blob/audit-logs/local-provider/packages/providers/upload-local/lib/index.js#L32)